README.md

```diff
@@ -176,8 +178,9 @@ datadog_config:
     scrub_args: true
     custom_sensitive_words: ['consul_token','dd_api_key']
 system_probe_config:
-  enabled: true
```

Comment:
So this line was removed based on comment https://github.com/DataDog/ansible-datadog/pull/326#discussion_r587487488, but maybe it would be better to leave it here with a note that it may turn on billed features? Providing this option without `enabled: true` is kind of pointless and misleading, as noted in https://github.com/DataDog/ansible-datadog/issues/330.

Reply:
From our convo: this should not be set directly. However, that'll require updating the Linux role to not crash when this option is not there (by adding an `if 'enabled' in system_probe_config`) and to support the `network_config['enabled']: true` option.